ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.BrayfloxsLongstop,
+  triggers: [
+    {
+      id: 'Brayflox Normal Numbing Breath',
+      netRegex: NetRegexes.startsUsing({ id: '1FA', source: 'Great Yellow Pelican' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Esuna Pelican Poison
+      // This covers cases were multiple people get the poison, including yourself.
+      id: 'Brayflox Normal Pelican Poison Healer',
+      netRegex: NetRegexes.gainsEffect({ effectId: '12' }),
+      alertText: function(data, matches) {
+        if (matches.target != data.me && data.role == 'healer') {
+          return {
+            en: 'Esuna Poison on ' + data.ShortName(matches.target),
+          };
+        }
+        if (matches.target == data.me && data.role == 'healer') {
+          return {
+            en: 'Esuna Your Poison',
+          };
+        }
+      },

```

Comment:
Maybe rewrite this with a single-line condition like `condition: (data) => data.role === 'healer',`? (Also note the strict non-comparison here, not loose! Unless there's a specific reason for it, we're generally moving to use only strict comparison, and if there *is* a reason, consider rewriting to avoid it.)

```javascript
  condition: (data) => data.role === 'healer',
  alertText: function(data, matches) {
    if (matches.target !== data.me) {
      return {
        en: 'Esuna Poison on ' + data.ShortName(matches.target),
      };
    }
    return {
      en: 'Esuna Your Poison',
    };
  },

```